base/expvar.go

```diff
@@ -82,6 +82,10 @@ const (
 	StatKeyChannelCachePendingQueries = "chan_cache_pending_queries"
 	StatKeyNumSkippedSeqs             = "num_skipped_seqs"
 	StatKeyAbandonedSeqs              = "abandoned_seqs"
+	StatKeyMaxCached                  = "max_cached"
```

Comment:
Can we call these `max_seq_*`?

Reply:
My preference would be:
- high_seq_cached (for max_cached)
- high_seq_stable (for max_stable_cached)
- high_seq_feed (for dcp_max_received)

- skipped_seq_len (for StatKeyNumSkipped)
- pending_seq_len (for StatKeyNumPending)